amundsen_application/static/js/components/TableDetail/styles.scss

```diff
@@ -1,53 +1,42 @@
 @import 'variables';
 
 .table-detail {
-  margin: 40px auto 48px;
-  display: flex;
-  flex-direction: row;
-}
-
-/*
-  white-space: nowrap breaks without this width on the row
-*/
-.table-detail .row {
-  width: 100%;
-  margin: 0;
-}
```

Comment:
The `margin: 0` was adding an extra 30px of margin since by default bootstrap sets row as `margin: -15px` and add it back as padding in each `col-x` element. I also did some testing and found no issues with column descriptions overflowing. There must have been another cause that was fixed.